pkgs/http_profile/README.md

```diff
@@ -1,2 +1,68 @@
-An **experimental** package that allows HTTP clients outside of the Dart SDK
-to integrate with the DevTools Network tab.
+[![pub package](https://img.shields.io/pub/v/http_profile.svg)](https://pub.dev/packages/http_profile)
+[![package publisher](https://img.shields.io/pub/publisher/http_profile.svg)](https://pub.dev/packages/http_profile/publisher)
+
+A package that allows HTTP clients outside of the Dart SDK to integrate with
+the DevTools Network tab.
+
```

Comment:
Maybe:

**NOTE:** This package is mean for developers *implementing* HTTP clients, not developers *using* HTTP clients.

Reply:
Done